index.html

```diff
@@ -222,28 +243,29 @@ <h2>Extensions to the <dfn data-cite=
       <p>The <dfn>PerformanceEntryList</dfn> represents a sequence of
       <a>PerformanceEntry</a>, providing developers with all the convenience
       methods found on JavaScript arrays.</p>
-      <section>
-        <h2><dfn>getEntries()</dfn> method</h2>
-        <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
-        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
-        set to <a>relevant performance entry buffer</a>, and <var>name</var>
-        and <var>type</var> set to <code>null</code>.</p>
-      </section>
-      <section>
-        <h2><dfn>getEntriesByType()</dfn> method</h2>
-        <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
-        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
-        set to <a>relevant performance entry buffer</a>, <var>name</var> set to
-        <code>null</code>, and <var>type</var> set to `type`.</p>
-      </section>
-      <section>
-        <h2><dfn>getEntriesByName()</dfn> method</h2>
-        <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
-        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
-        set to <a>relevant performance entry buffer</a>, <var>name</var> set to
-        `name`, and <var>type</var> set to <code>null</code> if optional
-        `entryType` is omitted, and <var>type</var> set to `type`
-        otherwise.</p>
+        <section>
+          <h2><dfn>getEntries()</dfn> method</h2>
+          <p>Returns a <a>PerformanceEntryList</a> object returned by
+          <a>filter buffer map by name and type</a> algorithm with
+          <var>name</var> and <var>type</var> set to <code>null</code>.</p>
+        </section>
+        <section>
+          <h2><dfn>getEntriesByType()</dfn> method</h2>
+          <p>Returns a <a>PerformanceEntryList</a> object returned by
+          <a>filter buffer map by name and type</a> algorithm with
+          <var>name</var> set to <code>null</code>, and
+          <var>type</var> set to the method's input <code>type</code>
```

Comment:
Since you're fixing this, I don't think we need to always specify the name of the parameters when calling an algorithm. For example instead of
algorithm x with name1 set to var1 and name2 set to var2
You can just say something like:
algorithm x passing var1 and var2 as inputs.

The order of the inputs should match the order in the algorithm description.

Reply:
While that's not a requirement, I think it improves clarity, so maybe leave that in?